terraform/kubernetes.tf

```diff
@@ -53,6 +53,10 @@ resource "azurerm_kubernetes_cluster" "aks" {
     enabled                       = "true"
   }
 
+  # identity {
+  #   type = "SystemAssigned"
+  # } 
+
```

Comment:
This has been added in then commented out. Is this correct?

Reply:
Yep, may be needed for key vault integration - comment added